linkerd/sample_apps.go

```diff
@@ -0,0 +1,32 @@
+package linkerd
+
+import (
+	"fmt"
+
+	"github.com/layer5io/meshery-adapter-library/adapter"
+	"github.com/layer5io/meshery-adapter-library/status"
+	"github.com/layer5io/meshkit/utils"
+)
+
+func (linkerd *Linkerd) installSampleApp(del bool, templates []adapter.Template) (string, error) {
+	linkerd.Log.Info(fmt.Sprintf("Requested action is delete: %v", del))
+	st := status.Installing
+
+	if del {
+		st = status.Removing
+	}
+
+	for _, template := range templates {
+		contents, err := utils.ReadRemoteFile(string(template))
```

Comment:
what if it is a local file? local files should be supported, if I'm not mistaken they are not in this func.
the function could be renamed to e.g. ReadFileResource(...), supporting file://, http://, gs:// ... that would be useful :-)

Reply:
I second that, working on these changes @mgfeller

Reply:
Is this functionality not defined in MeshKit? It seems that MeshKit should provide standardized support for local and remote manifest retrieval.